assets/js/custom/Translation.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jquery */
+
+// eslint-disable-next-line no-unused-vars
+function Translation (hasDescription, hasCredit) {
+  const elementsToTranslate = [document.getElementById('name')]
+
+  if (hasDescription) {
+    elementsToTranslate.push(document.getElementById('description'))

```

Comment:
If you pass the content as parameters directly from `Twig`, there is no need to filter it from the HTML page.
Just evaluate the parameters if they are set and contain a value.

Reply:
> However, there is no need to change it.

change needed?

Reply:
No, I just wanted to mention it. It works perfectly fine as it is - so let's keep it. 👍